### ISSUE TITLE
[BUG] : keeps removing versions when i download them

### ISSUE THREAD

**Issue by Zacbary** [Zagrios/bs-manager#591]

**Comment by silentrald** [comment]
Can you post your logs.

![image](https://github.com/user-attachments/assets/24aad976-c4d9-4478-af0e-4bc6fd4fcaf0)

Can you also add some more context of the problem such as your system specs.

**Comment by Crul0** [comment]
hav9ng the same issue

**Comment by silentrald** [comment]
@Crul0 can you post your logs?

**Comment by Crul0** [comment]
[2024-10-8-v1.4.8.log](https://github.com/user-attachments/files/17299913/2024-10-8-v1.4.8.log)

**Comment by silentrald** [comment]
@Crul0 There's only 1 unexpected error in the logs but other than that, I don't see anything inherently wrong in the logs 🤔. Although here are some few suggestion that might hopefully fix your issue.

* Try downloading the pre-release version of BSManager (might have been resolved there) which should be on this link [1.5.0-alpha.4](https://github.com/Zagrios/bs-manager/releases/tag/v1.5.0-alpha.4)
* Try deleting the extra versions in your BSInstances folder. There's 1 version that seems to be incomplete or corrupted so might try to delete the duplicates [the folders with (1) (2) (3)].
* Move your old installed version to another folder where BSM won't see it then try to install a new version.

If nothing works, might need to relook into your new logs if anything new error log pops up.

```
[2024-10-08 18:38:21.750] [info]  getVersionOfBSFolder C:\Users\private\BSManager\BSInstances\1.37.1 undefined
[2024-10-08 18:38:21.764] [info]  getVersionOfBSFolder C:\Users\private\BSManager\BSInstances\1.37.1 (1) undefined
This one is unexpected -> [2024-10-08 18:38:21.776] [info]  unable to get version from globalgamemanagers file C:\Users\private\BSManager\BSInstances\1.37.1 (1)\Beat Saber_Data\globalgamemanagers
[2024-10-08 18:38:21.784] [info]  getVersionOfBSFolder C:\Users\private\BSManager\BSInstances\1.37.1 (2) undefined
[2024-10-08 18:38:21.793] [info]  getVersionOfBSFolder C:\Users\private\BSManager\BSInstances\1.37.1 (3) undefined
[2024-10-08 18:38:21.807] [info]  getVersionOfBSFolder C:\Users\private\BSManager\BSInstances\Beat Saber undefined
```

**Comment by Crul0** [comment]
when i run it as admin it works fine

**Comment by silentrald** [comment]
Yeah, seems that there are permission problems when reading some files/folders. Should be a good workaround to run BSM as admin for now.
